### PR TITLE
light: add page

### PR DESCRIPTION
**Pull request by mstruebing**
Light is a CLI utility to control the backlight, see here: https://github.com/haikarainen/light

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/light.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ligth
```
Comment (agnivade) [inline pages/linux/light.md]: typo

File: pages/linux/light.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+> CLI to control backlight.
+
+- Get the current backlight value in percent:
+
+`light`
+
+- Set the backlight value to percent:
```
Comment (agnivade) [inline pages/linux/light.md]: Perhaps `Set the backlight value to a given percent` is better.

File: pages/linux/light.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`light -S {{percent}}`
+
+- Reduce percentage to the current backlight value:
```
Comment (agnivade) [inline pages/linux/light.md]: I guess `Reduce given percentage from the current backlight value` makes more sense ?

**Comment by mstruebing** [comment]
@agnivade Is this how you like it? :)

**Comment by agnivade** [comment]
I would have given actual values in the examples. That makes it much more clear. 

This text "given percentage from/to ... in percent" repeats percent twice. I guess mentioning an actual value would allow us to be a bit more relaxed in the description.

Does that make sense ? Sorry, if I am being too nitpicky. :sweat:

**Comment by mstruebing** [comment]
Now, with specific values. Is this how you like it?

File: pages/linux/light.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# light
+
+> CLI to control backlight.
```
Comment (sbrl) [inline pages/linux/light.md]: Perhaps ...`the backlight of your screen.`?

Comment (mstruebing) [reply pages/linux/light.md]: adjusted

**Comment by sbrl** [comment]
Man, I keep forgetting about wrapping things in tokens. I need to try and remember.... :P

**Pull request merged**